### DISCUSSION TITLE
MacOS / Safari 14.1

### DISCUSSION THREAD

**Discussion by craigtsmith** [MasterKale/SimpleWebAuthn#127] [Troubleshooting]
Does this support MacOS / Safari 14.1? I can get it to work on Chrome, but not Safari, I had though Safari 14 introduced webauthn support using TouchID, am I missing something?

It seems every single demo of webauthn I've tried doesn't work on Mac OS / Safari 14.1, but everything I _read_ seems to indicate it should work!

**Comment by craigtsmith** [comment]
I think I _may_ have just answered my own question - it _seems_ that webauthn / touchid in safari is only for Big Sur (Mac OS 11) & not Catalina (Mac OS 10.15) is this the case?

**Comment by MasterKale** [comment]
Touch ID on macOS is [only supported in Big Sur](https://webkit.org/blog/11312/meet-face-id-and-touch-id-for-the-web/):

> - Face ID and Touch ID for the web is available in Safari, SFSafariViewController and ASWebAuthenticationSession on iOS 14, iPadOS 14 and macOS Big Sur. For macOS, Safari 14 with downlevel OS will not get this feature because the attestation relies on a new system framework.

Before then only security keys were usable.

Once you get on Big Sur SimpleWebAuthn will support it just fine 😀
